### ISSUE TITLE
paper-textarea label and descriotion are not accessible to screen readers on Chrome

### ISSUE THREAD

**Issue by hcarmona** [PolymerElements/paper-input#614]
### Description

It looks like the textarea's aria-labelledby is populated with the id of the label, however, the label is not in the same shadow dom as the textarea, so this is not accessible in Chrome. This means that screen readers are not finding the label. (Same is true for aria-describedby)

### Expected outcome

Screen readers should be able to read the label

### Actual outcome

Screen readers can't read the label of a paper-textarea

### Live Demo
http://jsbin.com/fepepop/edit?html,output
(Taken from Issue 478)

### Steps to reproduce

1. Put a paper-textarea in the page
2. Open the page in Chrome
3. Use a screen reader to navigate to teh paper-textarea

### Browsers Affected

- [ Fail ] Chrome
- [ Pass ] Firefox
- [ Pass ] Safari 11
- [ Not Tested ] Edge
- [ Not Tested ] IE 11
